packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -568,7 +568,9 @@ function VM-Install-With-Installer {
         [Parameter(Mandatory=$false)]
         [bool] $consoleApp=$false,
         [Parameter(Mandatory=$false)]
-        [string] $arguments = ""
+        [string] $arguments = "",
+        [Parameter(Mandatory=$false)]
+        [string] $iconLocation
```

Comment:
when is this new parameter used? It looks like it is overwritten (without checking if it is set)

Reply:
It is used for `VM-Install-Shortcut` at the bottom and is checked with: https://github.com/mandiant/VM-Packages/blob/c2ca59576b9711f089b67d1d7ee751b997980879/packages/common.vm/tools/vm.common/vm.common.psm1#L637C9-L637C66

The only place it is potentially overwritten is if it is an `MSI` package, since I opted to go the route of checking the installer path as a potentially more guaranteed icon location. I can move the MSI part into the `iconLocation` check instead, but I will still need to check if `iconLocation` is set after the `MSI` piece again, just in case it isn't found, which might make it look a little sloppy.